### PR TITLE
story(ccls-2069) amend application on proceeding change

### PR DESCRIPTION
**Pull request by PhilDigitalJustice**

### CODE REVIEW

File: src/main/java/uk/gov/laa/ccms/caab/service/ApplicationService.java [hunk 1]
```diff
@@ -233,6 +234,13 @@ public List<BaseApplication> getCases(
     return searchResults;
   }
 
+  public Mono<Void> patchApplication(
```
Comment (porritta) [inline src/main/java/uk/gov/laa/ccms/caab/service/ApplicationService.java:237]: I know it's just a one-liner, but could do with a javadoc.

**Review by porritta — CHANGES_REQUESTED** [review]
Just the one minor comment

File: src/main/java/uk/gov/laa/ccms/caab/service/ApplicationService.java (hunk 1 above)
Comment (PhilDigitalJustice) [reply src/main/java/uk/gov/laa/ccms/caab/service/ApplicationService.java:246]: Amended

**Review by porritta — APPROVED** [review]
LGTM

**Pull request merged**